cmd/minishift/cmd/hostfolder/umount.go

```diff
@@ -17,24 +17,21 @@ limitations under the License.
 package hostfolder
 
 import (
-	"fmt"
-
 	"github.com/docker/machine/libmachine"
 	"github.com/minishift/minishift/cmd/minishift/cmd/util"
 	"github.com/minishift/minishift/cmd/minishift/state"
 	"github.com/minishift/minishift/pkg/minikube/constants"
-	hostfolderActions "github.com/minishift/minishift/pkg/minishift/hostfolder"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/cobra"
 )
 
-var hostfolderUmountCmd = &cobra.Command{
-	Use:   "umount HOSTFOLDER_NAME",
-	Short: "Unmount a host folder from the running OpenShift cluster.",
-	Long:  `Unmount a host folder from the running OpenShift cluster. This command does not remove the host folder definition or the host folder itself.`,
+var umountCmd = &cobra.Command{
+	Use:   "umount HOST_FOLDER_NAME",
+	Short: "Umount a host folder from the running Minishift VM.",
+	Long:  `Umount a host folder from the running Minishift VM. This command does not remove the host folder config or the host folder itself.`,
```

Comment:
Ditto

Reply:
@gbraad Didn't get your comment here. Are you pointing `umount` ?

Reply:
Ditto for use of VM again, while also to OpenShift cluster exist in this pr.